tools/build_cpp.py

```diff
@@ -63,9 +63,8 @@ def main(prefix='install', build_dir='build', source_dir='.', caching=False):
                 '-DCMAKE_OSX_DEPLOYMENT_TARGET':
                 osxversion,
                 '-DCMAKE_OSX_SYSROOT':
-                os.path.join('/Applications', 'Xcode.app', 'Contents',
-                             'Developer', 'Platforms', 'MacOSX.platform',
-                             'Developer', 'SDKs',
+                os.path.join('/Applications', 'Xcode.app', 'Contents', 'Developer',
```

Comment:
This long path was removed entirely from the `scippneutron` PR. Can we remove it here also?

Reply:
Done